### ISSUE TITLE
sep is NoneType

### ISSUE THREAD

**Issue by borgchick** [izderadicka/imap_detach#7]
Thank you for making this. All the other imap clients I tried working with somehow all ended not working with my particular imap server. Yours worked fine, after this very minor tweak to utils.py:

```
106         if sep and sep != '/':
107             name=name.replace(sep, '/')

```
For me, sep was None, so line 107 would throw an exception.  With my change to line 106, the rest of your code works great!

**Comment by izderadicka** [comment]
Can you submit as PR? Thanks.

**Comment by izderadicka** [comment]
Fixed by PR #8
